l10n_eu_product_adr/models/product_dangerous.py

```diff
@@ -1,13 +1,15 @@
 # Copyright 2019 Iryna Vyshnevska (Camptocamp)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class ProductDangerousClass(models.Model):
     _name = "product.dangerous.class"
     _description = "Product Dangerous Class"
+    _rec_name = "name_full"
 
     name = fields.Char(required=True, translate=True)
+    name_full = fields.Char(compute="_compute_name_full")
```

Comment:
I would named it `complete_name` like in:

- [`stock.location`](https://github.com/odoo/odoo/blob/13.0/addons/stock/models/stock_location.py#L30)
- [`product.category`](https://github.com/odoo/odoo/blob/aff7694c56185212c17ceaf358ad99b3d14dee8b/addons/product/models/product.py#L27)
- [`analytic.account.group`](https://github.com/odoo/odoo/blob/aff7694c56185212c17ceaf358ad99b3d14dee8b/addons/analytic/models/analytic_account.py#L46)
- `ir.ui.menu`
- `ir.model.fields`

`name_full` is not used at all in Odoo standard.

Reply:
Yes and changed.